app/models/rook.rb

```diff
@@ -1,6 +1,6 @@
 class Rook < Piece
   def valid_move?(dest_x, dest_y)
-    dest_piece = game.pieces
+    dest_piece = self.game.pieces
```

Comment:
not necessary, forgot I put that in there while testing....won't hurt either i suppose